style.css

```diff
@@ -0,0 +1,129 @@
+body {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  background-color: gray;
+}
+
+.mobmenu {
+  display: none;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.blue {
+  color: #6070ff;
+}
+
+.space {
+  margin: 0 5%;
+}
+
+.menu {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.menu a {
+  text-decoration: none;
+  color: gray;
+}
+
+.onhover:hover {
+  text-shadow: 0 0 0.15em #6070ff;
+  opacity: 0.5;
+}
+
+.pressed:active {
+  color: #6070ff;
+}
+
+.round {
+  border-bottom-left-radius: 80px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-color: #fff;
+}
+
+/* flex container */
+.toolbar {
+  display: flex;
+  align-items: center;
+  padding-top: 5%;
+}
+
+.toolbar ul {
+  margin-right: 5%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.liststyle {
+  list-style: none;
+}
+
+/* flex item */
+.toolbar li {
+  flex: 1;
+  margin: 0 10px;
+}
+
+.toolbar div {
+  flex: 2;
+}
+
+.wide {
+  max-width: 920px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  background-image: url('Img/Headerbg@2x.png');
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
- [ ] Your portfolio is looking great, well done! Let's adjust some details to make it more similar to the Figma template:

|Template|Your design|
|---|---|
| ![Captura de pantalla 2022-04-21 a la(s) 7 13 43 p  m](https://user-images.githubusercontent.com/83717285/164571490-ffd935b6-60a6-43e3-b7ec-d0bfabcf1299.png) | ![Captura de pantalla 2022-04-21 a la(s) 7 13 59 p  m](https://user-images.githubusercontent.com/83717285/164571513-10e14aa2-2c62-4b43-b3e6-094e417ff702.png) |

- Let's adjust the background size which is looking fairly different from the template.
- I would recommend removing the gray margin surrounding your design as this is not part of the template.
- It would be better to increase the text size of the main title and reduce the text size of the description.